cfme/intelligence/reports/ui_elements.py

```diff
@@ -608,6 +612,7 @@ def combo(self):
 
 @fill.method((DashboardWidgetSelector, Sequence))
 def _fill_dws_seq(dws, seq):
+    # Doesn't work now because of bz 1287767
```

Comment:
Yes, it should work after bug resolving, of course it is possible that we will need to correct the locator